src/FINDOLOGIC/Api/Validators/ParameterValidator.php

```diff
@@ -13,7 +14,7 @@ public function __construct(array $data = [], array $fields = [], $lang = null,
         $this->addInstanceRule('shopkey', function ($field, $value) {
             return (is_string($value) && preg_match('/^[A-F0-9]{32}$/', $value));
         }, self::ERROR_DEFAULT);
-        $this->addInstanceRule('revision', function ($field, $value) {
+        $this->addInstanceRule('version', function ($field, $value) {
```

Comment:
Isn't the parameter still called `revision`?

Reply:
Yes but this is the schema on how it's validated. This parameter should be a version number.